include/nfc/ndef/nfc_uri.rst

```diff
@@ -0,0 +1,64 @@
+.. _nfc_uri:
+
+URI messages
+############
+
+URI messages contain exactly one URI record, which in turn contains an address that an NFC polling device can open.
+In the most typical use case, the URI record contains a web address like "http\://www.nordicsemi.com" that the polling device opens in a web browser.
+
+URI records consist of a URI field (the actual address) and a URI identifier that specifies the protocol of the URI and is prepended to the URI.
+See :cpp:enum:`nfc_uri_id` for the available protocols.
+
+The :ref:`nfc_uri_record` module provides functions for creating the record, and the :ref:`nfc_uri_msg` module provides functions for creating and encoding the message.
+
+The following code snippets show how to generate a URI message.
+First, define the URI string and create a buffer for the message:
+
+.. code-block:: c
+
+   static const u8_t m_url[] =
```

Comment:
I've tried to update the code blocks in this file - but please check if they are correct.

Reply:
One small fix - otherwise looks ok